### DISCUSSION TITLE
Zepp Watchface Maker editable components

### DISCUSSION THREAD

**Discussion by dtreed85** [zepp-health/zeppos-forum#287] [Q&A] [answered]
Is there any good documentation, examples, or demonstrations of adding and configuring editable components in the Zepp Watchface Maker site?  They have an example watchface that has them, but there is no explanations for anything that I can find anywhere, and I can't get their "example 3" watchface to work.

**Comment by silver-zepp** [comment]
Have you seen these videos? @dtreed85 
[Create Watch Face Project](https://www.youtube.com/watch?v=vIJz0925gqA)
[Adding a Date and Week to the Watch Face](https://youtu.be/luCC6byMxK8)
[Adding a Background and Time to the Watch Face](https://www.youtube.com/watch?v=3Vj2UUgWyTc)

And here's [Watch Face Documentation](https://docs.zepp.com/docs/watchface/watchface-quick-start/)

Besides that, I'm afraid there are not many watch face resources.

**Reply by dtreed85** [reply]
Thanks for responding.  Appreciate it.

Yes I've watch all the relevant YouTube videos from "Zepp Health Developer" I believe.  They were helpful but I have yet to find anything for "editable components".  I've subscribed to that YouTube channel along with "Amazfit Global" too.  I've also been through the watch face documentation multiple times and it's useful, just not for editable components.  :(

Thanks again.  Still hopeful...

**Reply by silver-zepp** [reply]
Maybe @SashaCX75 can recommend some useful resources?

**Comment by SashaCX75** [comment, marked as answer]
I rarely use Zepp Watchface Maker
It seems in creating editable the name of all parameters is pretty clear. The main problem is to prepare all the graphics.
1. First of all you need to decide on the design. Determine how many editable zones will be and what elements will be in them.
2. In the "Component Mask" section specify the mask for the selection mode. Usually it is a semi-transparent file where areas with editable components remain unshaded.
3. In the "Editable Component" section, enable and configure the required components.
4. In the "Editable Component" section itself, adjust the position and size of the editable area, as well as the preview for the selected elements. Selects the image that highlights the zone in active and inactive states. Selects the image and its coordinates for the text hint, Specifies the size of the field for the text hint.
5. If there are several editable zones, another section "Editable Component" is created and customised.

**Reply by dtreed85** [reply]
Thanks for the help.   I appreciate it.  That is basically what I though, but just not clear on the tip thing or the "Label Image".  Unless that is for the tip?  I'm probably just doing something wrong.

Just started checking out your AmazFit Watchface Editor a few days ago since I have a T-Rex 2.

Thanks again.

**Reply by dtreed85** [reply]
Found the issue.  Even though the "Image not selected" image isn't marked as required, the editable component started working after I added one.

**Comment by SashaCX75** [comment]
"Label Image" is the background image for the tooltip (name of the selected item). Usually a light coloured rectangle with rounded edges.
